Ginger/Ginger/UserControlsLib/TextEditor/ValueExpression/ValueExpressionEditor.cs

```diff
@@ -41,7 +41,7 @@ public class ValueExpressionEditor : TextEditorBase
 
         private Page p=null;
 
-        public static readonly Regex rxVarFormulaParams = new Regex(@"[A-Za-z]*=[ A-Za-z0-9]*", RegexOptions.Compiled);
+        public static readonly Regex rxVarFormulaParams = new Regex(@"[A-Za-z]*=[ A-Za-z0-9!@#$%^&*_+?/><-]*", RegexOptions.Compiled);
```

Comment:
The fact that we missed to support existing variable in unit test (with having "?") making me worried that we will break users existing flows due to other things we missing, let's discuss this on Sunday to see if we can minimize the risk.
also we'll have a talk with Ginger team on that